### PR TITLE
Dream cakes business site - Ingela Löfgren

### PR DESCRIPTION
**Pull request by IngelaL**

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,30 +1,254 @@
...
+      required
+       ></textarea>
+    </div>
+
+      <label>How many pieces of cake?</label>
+
+    <div class="form-group">
+      <input 
+      type="radio" 
+      id="8"
```
Comment (EmOlsz) [inline code/index.html:103]: You did connection `input`-`form` very well, but you should avoid naming your `id`s by numbers only. You can add numbers to you `id` name, but they shouldn't start from it. So, you can name it `radio8` or `radio8element`, but not `8`, `8radio` or `8radioelement`

File: code/index.html [hunk 2]
```diff
@@ -1,30 +1,254 @@
...
+
+      <input 
+      type="radio" 
+      id="20" 
+      name="pieces" 
+      value="20">
+      <label for="20">20</label><br>
+    </div>
+
+      <label>Any food preferences?</label>
```
Comment (EmOlsz) [inline code/index.html:133]: If `label` is not connected to any `input` element, you can use `p` instead, it's better that way

File: code/index.html [hunk 3]
```diff
@@ -1,30 +1,254 @@
...
+
+    <div class="form-group">
+      <input 
+      type="checkbox" 
+      id="Glutenfree"
```
Comment (EmOlsz) [inline code/index.html:138]: We should also avoid naming `id`s uppercase

File: code/index.html [hunk 4]
```diff
@@ -1,30 +1,254 @@
...
+    </div>
+
+      <label>Any food preferences?</label>
+
+    <div class="form-group">
+      <input 
+      type="checkbox" 
+      id="Glutenfree" 
+      name="foodallergies" 
+      value="Glutenfree">
```
Comment (EmOlsz) [inline code/index.html:140]: Same goes for `value` attribute. Basically, all code that is not visible for user (so, is visible only for computer), should be lowercase

File: code/index.html [hunk 5]
```diff
@@ -1,30 +1,254 @@
...
+
+      <input
+      type="checkbox" 
+      id="Vegan" 
+      name="foodallergies" 
+      value="Vegan">
+      <label for="allergi3">Vegan</label><br>
+    </div>
+
+      <button>Submit</button>
```
Comment (EmOlsz) [inline code/index.html:158]: Great job with form, you just forgot to add `type="submit"` attribute

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,211 @@
+
+body {
+ margin: 0px;
```
Comment (EmOlsz) [inline code/style.css:3]: If value is `0px`, we can omit `px` and just type `0`

File: code/style.css [hunk 7]
```diff
@@ -1,7 +1,211 @@
...
+ font-family: monospace;
+ background: seashell;
+}
+
+ /* Header */
+
+header {
+background-size: cover;
```
Comment (EmOlsz) [inline code/style.css:11]: some wrong indentation

File: code/style.css [hunk 8]
```diff
@@ -1,7 +1,211 @@
...
+  margin: 30px;
+  padding: 10px;
+  border: transparent;
+  border-radius: 50px;
+  box-shadow: 0 20px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
+  background-color: transparent;
+}
+
+button:hover {
+  transform: scale(0.9, 0.9);
```
Comment (EmOlsz) [inline code/style.css:47]: Very nice Ingela :)

File: code/style.css [hunk 9]
```diff
@@ -1,7 +1,211 @@
...
+.active + .answer {
+  display: block;
+}
+
+.active, .question:hover {
+  background-color: #CC95C0; 
+}
+
+.question:after {
+  content: '\02795'; /* Unicode character for "plus" sign (+) */
```
Comment (EmOlsz) [inline code/style.css:129]: 👍

**Review by EmOlsz — APPROVED** [review]
Well done Ingela, congratulations! I'm happy to see responsive video and working form and accordion. Moreover, I notice you've added different accordion icons depending on whether they are open or not, moreover form has some validators inside. You `display` property knowledge looks solid, that's very important (especially flexbox one). Lastly, you know how to create radio- and checkbox-groups. Good job :)

Nevertheless, I would like you to work on couple things I noticed during code review. Those are :
1. Sometimes you forgot to check your indentation - please, remember about that
2. You seem to add uppercase names from time to time - as I mentioned in one comment in code, we should rarely use them

**Comment by IngelaL** [comment]
Thanks a lot @EmOlsz for great feedback. Your comments help a lot for future projects!

**Pull request closed without merging**